uniflow/flow/config.py

```diff
@@ -255,35 +429,27 @@ class RaterForGeneratedAnswerConfig(RaterConfig):
             "strong reject": -2.0,
```

Comment:
Same as above. This is a very nice example. However, this long token length will cause a high cost. I would suggest to write an example with only 'accept, equivalent, and reject` to reduce cost.